tests/unit/commands/test_borgmatic.py

```diff
@@ -422,6 +422,39 @@ def test_run_actions_runs_rcreate():
         )
     )
 
+def test_run_actions_adds_log_file_to_hook_context():
+    flexmock(module).should_receive('add_custom_log_levels')
+    flexmock(module.command).should_receive('execute_hook')
+    flexmock(borgmatic.actions.create).should_receive('run_create').with_args(
+        config_filename=flexmock(),
+        repository={'path': 'repo'},
+        location={'repositories': []},
+        storage=flexmock(),
+        hooks={},
+        hook_context={'log_file': 'foo'},
+        local_borg_version=flexmock(),
+        create_arguments=flexmock(),
+        global_arguments=flexmock(dry_run=False, log_file='foo'),
+        dry_run_label='',
+        local_path=flexmock(),
+        remote_path=flexmock(),
+    ).once()
```

Comment:
FYI there's a cool trick you can do with Flexmock if there's an argument you don't really care about setting an expectation for. Just set its value inside a `.with_args()` call to `object`. That effectively tells flexmock "any object will do here; I don't really care what the value is".

I'm not saying you have to do that here. Just letting you know because it often saves on boilerplate.